File: LeetCode-Programs/Check if Array Is Sorted and Rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        count, size=0, len(nums)
        #for i in nums:
          #  if i>i+1:
        for i in range(len(nums)-1):
            if nums[i]>nums[i+1]:
                count=count+1
        if count==1 and nums[0]>=nums[size-1]:
            return True
        elif count>=1:
            return False
        return True

File: LeetCode-Programs/Check if Array Is Sorted and Rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int  count=0,size=nums.size();
        for(int i=1;i<nums.size();i++)
        {
            if(nums[i-1]>nums[i])
                count++;
        }
        if(count==1 && nums[0]>=nums[size-1])
            return true;
        else if(count>=1)
            return false;
        return true;
    }
};
